ui/component/viewers/videoViewer/view.jsx

```diff
@@ -189,7 +213,12 @@ function VideoViewer(props: Props) {
         }
       });
       player.on('ratechange', () => {
-        if (player) {
+        const HAVE_NOTHING = 0; // https://docs.videojs.com/player#readyState
+        if (player && player.readyState() !== HAVE_NOTHING) {
+          // The playbackRate occasionally resets to 1, typically when loading a fresh video or when 'src' changes.
+          // Videojs says it's a browser quirk (https://github.com/videojs/video.js/issues/2516).
+          // [x] Don't update 'videoPlaybackRate' in this scenario.
+          // [ ] Ideally, the controlBar should be hidden to prevent users from changing the rate while loading.
           setVideoPlaybackRate(player.playbackRate());
         }
```

Comment:
Can we change the scope of this PR to just address the improvement to the playback rate value?

I believe the issue related the the volume slider is caused elsewhere, and it'd be easier to isolate any regression if the PR is smaller. 

I have some refactoring done in the other lines modified that causes some conflicts as well.